exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/writer/TestCorruptParquetDateCorrection.java

```diff
@@ -59,6 +61,7 @@
  * Use of this option is assumed to be extremely unlikely, but it is included
  * for completeness.
  */
+@Category(ParquetTest.class)
```

Comment:
Secondary? Would not seem that this test need to be run on each build...

Reply:
Done